magento/magento2
# Activated "Pending Reviews" menu item when merchant opens 'Pending Reviews' section

### Description (*)

This pull request will activate the "Pending Reviews" menu item when merchant opens admin path 'Marketing > User Content > Pending Reviews'
### Fixed Issues (if relevant)

1. magento/magento2#26229 >: Active menu is not set when opening admin path Marketing > User Content > Pending Reviews

### Manual testing scenarios (*)

1. Go to admin path 'Marketing > User Content > Pending Reviews' & notice that 'Marketing' menu item is now shown as active

### Questions or comments


### Contribution checklist (*)
